### ISSUE TITLE
Updates for installation

### ISSUE THREAD

**Issue by GeoffAtHome** [manzanotti/geniushub-client#87]
While attempting to build on a new system (Windows) I am capturing here some issues I have run into.
 ```
python .\setup.py installTraceback (most recent call last):
File "setup.py", line 8, in <module>
from setuptools import find_packages, setup
ModuleNotFoundError: No module named 'setuptools'
```
Fix: `pip install setuptools`

```
python .\setup.py install
Traceback (most recent call last):
setup.py", line 15, in <module>
 VERSION = os.environ["GITHUB_REF_NAME"]
   ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "<frozen os>", line 685, in __getitem__
KeyError: 'GITHUB_REF_NAME'
```
Could not solve this using Windows POWERSHELL. Ended up using `pip install aiohttp`. This allowed me to run the client locally.

**Comment by manzanotti** [comment]
@GeoffAtHome Funnily enough, I was setting up the project on my Windows laptop this weekend, and had all sorts of issues!

I also had to install aoihttp, which required me to install the Visual Studio C++ build tools!

I also had issues getting the pre-commit hooks to work, which I solved by upgrading all the packages in the dev workflow.

I've put them into the new PR for moving zone properties into a new class, which I was going to ask you to review anyway!

On your issue, does the readme need updating?
